bot/cogs/bot.py

```diff
@@ -119,35 +138,59 @@ def codeblock_stripping(self, msg: str):
             on_cooldown = time.time() - self.channel_cooldowns[msg.channel.id] < 300
             if not on_cooldown or msg.channel.id == DEVTEST_CHANNEL:
                 try:
-                    content = self.codeblock_stripping(msg.content)
-                    if not content:
-                        return
-
-                    # Attempts to parse the message into an AST node.
-                    # Invalid Python code will raise a SyntaxError.
-                    tree = ast.parse(content)
-
-                    # Multiple lines of single words could be interpreted as expressions.
-                    # This check is to avoid all nodes being parsed as expressions.
-                    # (e.g. words over multiple lines)
-                    if not all(isinstance(node, ast.Expr) for node in tree.body):
-                        codeblock_tag = await self.bot.get_cog("Tags").get_tag_data("codeblock")
-
-                        if codeblock_tag == {}:
-                            log.warning(f"{msg.author} posted something that needed to be put inside Python "
-                                        "code blocks, but the 'codeblock' tag was not in the tags database!")
-                            return
+                    howto_embed = ""
+                    not_backticks = ["'''", "\"\"\"", "´´´"]
+                    python_syntax = any(
+                        msg.content[3:9].lower() == "python",
+                        msg.content[3:5].lower() == "py"
+                    )
+
+                    bad_ticks = msg.content[:3] in not_backticks
 
-                        log.debug(f"{msg.author} posted something that needed to be put inside python code blocks. "
-                                  "Sending the user some instructions.")
-                        howto = (f"Hey {msg.author.mention}!\n\n"
-                                 "I noticed you were trying to paste code into this channel.\n\n"
-                                 f"{codeblock_tag['tag_content']}")
+                    if bad_ticks and python_syntax:
```

Comment:
okay, this catches the case where it's an otherwise correct codeblock, but uses the wrong ticks. however, if this is a malformed codeblock with the wrong ticks, what then? Perhaps you could somehow change the bad ticks into good ones before you later execute the codeblock_stripping so that it will still give instructions on how to make a working codeblock?